scenes/game/game.tscn

```diff
@@ -675,6 +679,7 @@ text = "DEFEAT"
 horizontal_alignment = 1
 
 [node name="ChoicePopoutShowButton" type="Button" parent="."]
+visible = false
```

Comment:
I found this convenient to have shown in the editor, so would prefer not invisible